### DISCUSSION TITLE
Does Clustered Strongswan support IKEv1?

### DISCUSSION THREAD

**Discussion by felartu** [strongswan/strongswan#2273] [Q&A] [answered]
Hello,

I'm reading here about the Clustered Highly available strongswan option:
https://docs.strongswan.org/docs/5.9/features/highAvailability.html

I'm looking to implement this solution but I have IKEv1 and IKEv2 tunnels, does this solution support IKEv1 as well? In the document states IKEv2 only but I wanted to confirm.

Thanks,

**Comment by tobiasbrunner** [comment, marked as answer]
It does support IKEv1, but not sure how well it actually works. I wouldn't recommend using that protocol anyway anymore.
